### DISCUSSION TITLE
Dataset Aware Scheduling - execution_date option?

### DISCUSSION THREAD

**Discussion by Raul824** [apache/airflow#33571] [Q&A] [answered]
Hello All,

Just wanted to know is there any execution_date option in dataset aware scheduling similar to Trigger Dag Run Operator.

Use Case ->

I have a dag C which is being dataset triggered by Dag A.

Dag C: 
it has two task groups C_Task_Group_1 -> C_Task_Group_2 

C_Task_Group_1 is dependent on Dag A 
C_Task_Group_2 is Dependent on Dag B using external sensor with current date dependency.

Dag A is scheduled every afternoon 12, now if there is 12 hours delay in Dag A, it will trigger Dag C with Next Date which will break external Dag dependency. (have created a custom function for external dag dependency which checks the manual runs as well using the execution date)

Dag C execution date change due to any delay can be handled in TriggerDagRunOperator using execution_date parameter, so was just wondering, is there similar kind of option or feature available in Dataset Aware Scheduling?

**Comment by potiuk** [comment, marked as answer]
Nope. No such option for now
